.harness/inputset.yaml

```diff
@@ -0,0 +1,39 @@
+inputSet:
+  identifier: "inputset"
+  pipeline:
+    identifier: "nodejsstarterpipeline"
+    properties:
+      ci:
+        codebase:
+          build:
+            type: "branch"
+            spec:
+              branch: "main"
+    stages:
+    - stage:
+        identifier: "build_test_and_run"
+        type: "CI"
+        spec:
+          execution:
+            steps:
+            - step:
+                identifier: "build_and_push_image_to_docker"
+                type: "BuildAndPushDockerRegistry"
+                spec:
+                  connectorRef: "DhrubajyotiDocker"
+                  repo: "dhrubajyotichakraborty/nodejshelloworld"
```

Comment:
Do the users need to mention the forked repo? If yes, keep it more generic like `GITUSERNAME/FORKEDREPO`

Reply:
DONE